controllers/users.js

```diff
@@ -191,6 +192,30 @@ const getUsers = async (req, res) => {
       }
     }
 
+    const isDeparted = req.query.departed === "true";
+
+    if (isDeparted) {
+      if (!dev) {
+        return res.boom.notFound("Route not found");
+      }
+      try {
+        const result = await dataAccess.retrieveUsers({ query: req.query });
+        const departedUsers = await userService.getUsersWithIncompleteTasks(result.users);
+        if (departedUsers.length === 0) return res.status(204).send();
```

Comment:
it might be more appropriate to use 200 with an empty array, as you are still sending a response?

Reply:
For empty results we are returning a 204 status code, which signifies that the request was successful but there is no content to return.
The send() just sends empty response